CITATION.cff

```diff
@@ -4,11 +4,11 @@
 cff-version: 1.2.0
 title: Stockfish
 message: >-
-  If you use this software, please cite it using the
+  If you use cffinit, please cite it using the
```

Comment:
?

Reply:
Oops! I ment to say _Stockfish_ here. Thanks for pointing out my mistake, @vondele.

Reply:
It is fixed now in the latest commit.